scripts/test.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+var path = require('path'),
+		CLIEngine = require('eslint').CLIEngine,
+		eslintCli = new CLIEngine(),
+		karma = require('karma'),
+		testConfig = {configFile : path.join(__dirname, '../spec/karma.conf.js')};
```

Comment:
While we're at it, could we simply get rid of this script fully? Add separate NPM scripts for ESLint and use the default Karma CLI.

Then we'd only be left with scripts for building things, so we'd leave the files in the `build` directory.

Reply:
Good point.
This solution will have a tiny drawback - lack of custom console logs (like `Checking for JS errors...` before linting source code).
Personally, I'm okay with that, since logs still would be readable enough to find out which script succeeded / failed.
But there is always an option to log this info using bash commands.
What do you think about that?

Reply:
I think it's enough to have various steps in separate `npm` scripts — that way the relevant command will be printed before execusion so you know what's going on (e.g. `> npm run lint-spec`).

Reply:
Done